tests/Feature/Authentication/LoginTest.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Tests\Feature\Authentication;
+namespace Feature\Authentication;
```

Comment:
The namespace is does start with `Tests\` via `composer.json`.

Reply:
The IDE was yelling that it was not compliant. Is there something else we can fix to prevent that?

Reply:
Does it do that for all tests?

Does it go away if you do `composer dump`? What about invalidating PHPStorm's caches via the File menu?

Reply:
Yes, it does it for the other tests as well. Am afk so can’t check purging the phpstorm cache, but have definitely dumped composer.